src/VirtoCommerce.Platform.Web/wwwroot/js/common/directives/uiScroll.tpl.html

```diff
@@ -0,0 +1,11 @@
+<div class="form-group">
+<label class="form-label">{{label | translate}}</label>
+<div class="form-input">
+    <ui-select ng-model="selectedId" required on-select="setValue($item)">
+        <ui-select-match placeholder="{{placeholder | translate}}">{{list.length > 0 ? $select.selected.name : 'Loading...'}}</ui-select-match>
+        <ui-select-choices repeat="x.id as x in list | filter: { name: $select.search }" refresh="fetch($select)" when-scrolled="fetchNext($select)">
+            <span ng-bind-html="x.name | highlight: $select.search"></span>
```

Comment:
I think *id* is OK, but it will be good to have a way to change the display field instead of only *name*.

Reply:
Don't forget to raise the platform dependency version in modules...